src/main/java/it/gov/pagopa/payhub/activities/service/paymentsreporting/PaymentsReportingMapperService.java

```diff
@@ -0,0 +1,65 @@
+package it.gov.pagopa.payhub.activities.service.paymentsreporting;
+
+import it.gov.digitpa.schemas._2011.pagamenti.CtFlussoRiversamento;
+import it.gov.pagopa.payhub.activities.dto.IngestionFlowFileDTO;
+import it.gov.pagopa.payhub.activities.dto.paymentsreporting.PaymentsReportingDTO;
+import org.springframework.context.annotation.Lazy;
+import org.springframework.stereotype.Service;
+
+import java.sql.Date;
+import java.time.Instant;
+import java.util.List;
+
+/**
+ * Service class responsible for mapping flow data (`CtFlussoRiversamento`) and ingestion metadata
+ * (`IngestionFlowFileDTO`) into a list of `PaymentsReportingDTO` objects.
+ */
+@Lazy
+@Service
+public class PaymentsReportingMapperService {
+
+	/**
+	 * Maps a `CtFlussoRiversamento` object and an `IngestionFlowFileDTO` into a list of `PaymentsReportingDTO` objects.
+	 * Each `PaymentsReportingDTO` in the list corresponds to an individual payment within the flow.
+	 *
+	 * @param ctFlussoRiversamento the flow data object containing detailed information about the transaction flow,
+	 *                             including details for multiple individual payments.
+	 * @param ingestionFlowFileDTO the ingestion metadata containing information about the processing flow.
+	 * @return a list of `PaymentsReportingDTO` objects, one for each individual payment in the flow.
+	 */
+	public List<PaymentsReportingDTO> mapToDtoList(CtFlussoRiversamento ctFlussoRiversamento, IngestionFlowFileDTO ingestionFlowFileDTO) {
+
+		PaymentsReportingDTO.PaymentsReportingDTOBuilder builder = PaymentsReportingDTO.builder()
+			.creationDate(Instant.ofEpochMilli(System.currentTimeMillis()))
+			.lastUpdateDate(Instant.ofEpochMilli(System.currentTimeMillis()))
+			.acquiringDate(new Date(ingestionFlowFileDTO.getCreationDate().getTime()).toLocalDate())
```

Comment:
check if the field contains just date or date/time and fix Date to LocalDate/LocalDateTime conversion
https://www.baeldung.com/java-date-to-localdate-and-localdatetime

Reply:
it just a LocalDate